examples/litellm_models/LiteLLM_OpenAI_example.py

```diff
@@ -0,0 +1,66 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from colorama import Fore
+
+from camel.models.litellm_model import LiteLLMModel
+from camel.types import ModelType
+from camel.utils import print_text_animated
+
+
+def main(model_type=None) -> None:
+    # Define model type and configuration
+    model_type = ModelType.GPT_3_5_TURBO
+    model_config_dict = {
+        "temperature": 0.7,
+        "max_tokens": 150,
+        "stream": False  # Set to True to enable stream mode
```

Comment:
currently "stream": True is not supported with litellm due to the limitation of current camel design, maybe add this information as # NOTE into `class LiteLLMModel(BaseModelBackend)`

Reply:
thank you for pointing this out. I have added a note to the `LiteLLMModel` class.